docs/advanced_usage/advanced_config_file.rst

```diff
@@ -118,6 +118,78 @@ Relative paths in the sub-YAML file are then resolved **relative to the sub-YAML
     :language: yaml
     :lines: 28-37
 
+
+Sharing a communication channel between multiple auxiliaries
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In order to attach a single communication channel to multiple defined auxiliaries, it is sufficient
+to use the same channel name for all auxiliaries.
+
+Under the hood, ``pykiso`` will automatically create a :py:class:`~pykiso.lib.auxiliaries.proxy_auxiliary.ProxyAuxiliary`
+that will be the only auxiliary having direct access to the communication channel, along with a
```

Comment:
```suggestion
that will have exclusive access to the communication channel.
```

Reply:
Rephrased it according to your suggestions ;)